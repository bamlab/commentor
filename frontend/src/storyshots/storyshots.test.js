import initStoryshots, { Stories2SnapsConverter } from '@storybook/addon-storyshots';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';

/**
 * this is used to split shapshot to different files
 * @todo  Use screenshot instead of snapshot ( or at least mount, but absolute filename breaks our ci)
 *  */
initStoryshots({
  test: ({ story, context }) => {
    const converter = new Stories2SnapsConverter();
    const snapshotFilename = converter.getSnapshotFileName(context);
    const storyElement = story.render();

    const tree = render(storyElement);

    expect(toJson(tree)).toMatchSpecificSnapshot(snapshotFilename);
  },
  // other options here
});
