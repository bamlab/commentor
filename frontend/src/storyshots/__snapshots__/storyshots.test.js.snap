// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button disabled 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <button
      className="Button-sc-1age22d-0 gflbGX"
      disabled={true}
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Button label 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <button
      className="Button-sc-1age22d-0 kkHZDp"
    >
      Coucou
    </button>
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Button onClick 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <button
      className="Button-sc-1age22d-0 kkHZDp"
      onClick={[Function]}
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Button playground 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <button
      className="Button-sc-1age22d-0 gflbGX"
      disabled={true}
      onClick={[Function]}
    >
      Hello Storybook
    </button>
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Button with no props 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <button
      className="Button-sc-1age22d-0 kkHZDp"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input CheckBox 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="checkbox"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Color 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="color"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Date: 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="date"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Datetime-local 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="datetime-local"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Email 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="email"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input File 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="file"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Hidden 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="hidden"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Image 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="image"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Month 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="month"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Number 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="number"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Password 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="password"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Radio 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="radio"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Range 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="range"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Reset 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="reset"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Search 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="search"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Submit 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="submit"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Tel 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="tel"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Text 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="text"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Time 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="time"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Url 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="url"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input Week 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 itqMQt"
      type="week"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input hasError 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 hhQdbo"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Input playground 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <input
      className="Input-lswqho-0 hhQdbo"
      type="text"
    />
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots InputRow disabled 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div
      className="InputRowstyle__Row-jt7egi-2 cqfxRi"
    >
      <input
        className="Input-lswqho-0 itqMQt"
        disabled={true}
        name="Name"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value="value"
      />
    </div>
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots InputRow error 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div
      className="InputRowstyle__Row-jt7egi-2 cqfxRi"
    >
      <input
        className="Input-lswqho-0 hhQdbo"
        name="Name"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value="value"
      />
      <p
        className="InputRowstyle__Error-jt7egi-1 bhyGMm"
      >
        error message
      </p>
    </div>
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots InputRow field 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div
      className="InputRowstyle__Row-jt7egi-2 cqfxRi"
    >
      <input
        className="Input-lswqho-0 itqMQt"
        name="Name"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value="value"
      />
    </div>
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots InputRow label 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div
      className="InputRowstyle__Row-jt7egi-2 cqfxRi"
    >
      <label
        className="InputRowstyle__Label-jt7egi-0 iueVXB"
      >
        Label
      </label>
      <input
        className="Input-lswqho-0 itqMQt"
        name="Name"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value="value"
      />
    </div>
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots InputRow placeholder 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div
      className="InputRowstyle__Row-jt7egi-2 cqfxRi"
    >
      <input
        className="Input-lswqho-0 itqMQt"
        name="Name"
        onBlur={[Function]}
        onChange={[Function]}
        placeholder="placeholder here"
        type="text"
        value=""
      />
    </div>
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots InputRow playground 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div
      className="InputRowstyle__Row-jt7egi-2 cqfxRi"
    >
      <label
        className="InputRowstyle__Label-jt7egi-0 iueVXB"
      >
        example
      </label>
      <input
        className="Input-lswqho-0 hhQdbo"
        disabled={true}
        name="Name"
        onBlur={[Function]}
        onChange={[Function]}
        placeholder="placeholder here"
        type="text"
        value="value"
      />
      <p
        className="InputRowstyle__Error-jt7egi-1 bhyGMm"
      >
        error
      </p>
    </div>
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots InputRow type 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div>
      type = text
    </div>
    <div
      className="InputRowstyle__Row-jt7egi-2 cqfxRi"
    >
      <input
        className="Input-lswqho-0 itqMQt"
        name="Name"
        onBlur={[Function]}
        onChange={[Function]}
        type="text"
        value="value"
      />
    </div>
    <div>
      type = password
    </div>
    <div
      className="InputRowstyle__Row-jt7egi-2 cqfxRi"
    >
      <input
        className="Input-lswqho-0 itqMQt"
        name="Name"
        onBlur={[Function]}
        onChange={[Function]}
        type="password"
        value="value"
      />
    </div>
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Loader playground 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div
      className="Loaderstyle__LoaderWrapper-sc-1gq6nz9-0 cqtykQ"
    >
      <div
        className="Loaderstyle__LoaderContainer-sc-1gq6nz9-1 gKNWuY"
      >
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
      </div>
    </div>
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;

exports[`Storyshots Loader with no props 1`] = `
Array [
  <div
    style={
      Object {
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div
      className="Loaderstyle__LoaderWrapper-sc-1gq6nz9-0 cqtykQ"
    >
      <div
        className="Loaderstyle__LoaderContainer-sc-1gq6nz9-1 gKNWuY"
      >
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
        <div />
      </div>
    </div>
  </div>,
  <button
    className="info__show-button"
    onClick={[Function]}
    style={
      Object {
        "background": "#027ac5",
        "border": "none",
        "borderRadius": "0 0 0 5px",
        "color": "#fff",
        "cursor": "pointer",
        "display": "block",
        "fontFamily": "sans-serif",
        "fontSize": "12px",
        "padding": "5px 15px",
        "position": "fixed",
        "right": 0,
        "top": 0,
      }
    }
    type="button"
  >
    Show Info
  </button>,
]
`;
